Devscord.DiscordFramework/Framework/Commands/Services/BotCommandsRequestValueGetterService.cs

```diff
@@ -10,59 +10,73 @@ public class BotCommandsRequestValueGetterService
     {
         public object GetValueByName(string key, bool isList, DiscordRequest request, BotCommandTemplate template)
```

Comment:
mamy testy do IBotCommand, ale są dość biedne, sprawdzające jedynie podstawowe przypadkI
(no chyba że w międzyczasie ktoś coś tam dodał)

dobrze byłoby dodać do testów jak najwięcej przypadków
zarówno sprawdzających klasyczną obsługe IBotCommand jak i CustomCommands 

`powód:`
ten mechanizm nie dość że jest skomplikowany, to jeszcze często się zmienia
bardzo łatwo o wypadek, przez który naprawienie jednego szczegółu zepsuje 3/4 komend
mając odpowiednio dużo testów sprawdzających odpowiednio dużo przypadków
nie będzie sytuacji, w której ktoś przypadkiem zepsuł mechanizm a reszta tego nie zauważy 

no i też prościej sie testuje klikając jeden przycisk
który od razu przenosi cie do specyficznej sytuacji 
niż odpalając bota, pisząc coś na kanale itd

Reply:
dobrze, napiszę testy później, gdy dodamy to na mastera i w kolejnym zadaniu, będę robił te testy.

Reply:
spoko, tak lepiej, tylko przed ostatecznym mergem postaraj się jak najdokładniej przetestować swoje zmiany
ale to już przed samym mergem